docs/docs/cmd/spo/sitescript/sitescript-get.mdx

```diff
@@ -67,8 +78,8 @@ m365 spo sitescript get --id 2c1ba4c4-cd9b-4417-832f-92a34bc34b2a
   <TabItem value="CSV">
 
   ```csv
-  Id,IsSiteScriptPackage,Title,Version
-  43d4ffa0-c7ee-4a97-91d7-db27e5b62de5,false,Contoso,1
+  Content,Id,IsSiteScriptPackage,Title,Version
+  "{\"$schema\":\"https://developer.microsoft.com/json-schemas/sp/site-design-script-actions.schema.json\",\"actions\":[{\"verb\":\"activateSPFeature\",\"name\":\"SiteNotebook feature\",\"title\":\"SiteNotebook feature\",\"featureId\":\"f151bb39-7c3b-414f-bb36-6bf18872052f\",\"scope\":\"web\"}],\"bindata\":{},\"version\":1}",43d4ffa0-c7ee-4a97-91d7-db27e5b62de5,false,Contoso,1
```

Comment:
isn't this a bug 😮? I think by default we don't return complex objects to `csv` and `md` right?

Reply:
It's stringified json, that's why it shows.

Reply:
I've rechecked and now even updated the md output response. the md and csv output response was apparently not right in de docs. Because the Content property is a string option, it will show in the output of the command in those modes.

Reply:
that is quite unexpected. but well 🙂. since this is a string it goes pass our checks and is printed.
Thanks @martinlingstuyl for the double check 👍